components/interaction/speechComp/README.md

```diff
@@ -1,35 +1,66 @@
 # speechComp
-This component is used as a Text to Speech component. It depends on the festival package that can be installed on Ubuntu with :  
+
+The `speechComp` component is a Python wrapper implementation of many libraries for the Text-to-Speech(TTS) task. The underlying logic is to run bash command utilizing `festival` TTS library or online TTS Google service for different languages via Python environment. This component uses `Speech.idsl` interface to implement template functions for running TTS from a string argument.
+
+## Compiling and Installation
+
+### Resolving dependencies
+
+This section assumes the user has already installed the RoboComp core library and pulled Robolab's components according to this [README guide](https://github.com/robocomp/robocomp). There are two TTS services:
+
+#### festival
+
+The current implementation provides two languages for TTS:
+
+- **English**
+To use English TTS, the user must install `festival` library, which is already available in Ubuntu PPA:
+
 ```bash
 sudo apt install festival
 ```
 
-The default languaje for festival is configured for English speech. If you want an Spanish speech you can download the Junta de Andalucía generated that can be found in this repository on github:
-https://github.com/guadalinex-archive/hispavoces/find/master
+- **Spanish**
+
+The default language of `festival` is English. If the user would like Spanish TTS, the user can download the Junta de Andalucía's synthetic voices that can be found in this [Github repository](https://github.com/guadalinex-archive/hispavoces/find/master).
 
-You can download the last festvox-sflpc16k version .deb package and install with:
+The user should download the latest `festvox-sflpc16k_1.0-1_all.deb` package and install with:
 ```bash
 sudo dpkg -i festvox-sflpc16k_1.0-1_all.deb
 ```
-You can test your installation with this command:
+
+and test the installation with this command:
 ```bash
 echo "Robocomp es el mejor software de robótica de Extremadura" | iconv -f utf-8 -t iso-8859-1 | festival --tts --language spanish
 ```
 
-## Configuration parameters
-As any other component,**`speech`** needs a configuration file to start. In
+#### Google TTS
+For the online Google TTS service, there are many more languages to choose from. Please install the `gTTS` library:
+
+```
+pip install gTTS
+```
 
-    etc/config
+### Compile the component
 
-you can find an example of a configuration file. You will have to set the Speech.Endpoint port to an available one in your deployment:
+Finally, we can compile `speechComp` to start using the component:
```

Comment:
Can you test this? I think this component doesn't need to be compiled.

Reply:
Yes, it does not require the comp to be compiled. I will fix that